### PR TITLE
[FIX] product_supplierinfo_for_customer: Get incorrect customerinfo for customer product price evaluation

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
@carlosdauden @Tardo review please

### CODE REVIEW

File: product_supplierinfo_for_customer/tests/test_product_supplierinfo_for_customer.py [hunk 1]
```diff
@@ -54,14 +54,14 @@ def _create_partnerinfo(cls, supplierinfo_type, partner, product):
             'price': 100.0,
         })
 
-    def test_default_get(self):
+    def _test_default_get(self):
```
Comment (MiquelRForgeFlow) [inline product_supplierinfo_for_customer/tests/test_product_supplierinfo_for_customer.py]: why putting the `_`?

Comment (sergio-teruel) [reply product_supplierinfo_for_customer/tests/test_product_supplierinfo_for_customer.py]: Upppsssss sorry

Comment (sergio-teruel) [reply product_supplierinfo_for_customer/tests/test_product_supplierinfo_for_customer.py]: REady

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Review by Tardo — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
Thanks for the extra test for avoiding regressions

**Pull request merged**